### PR TITLE
Create "Shared Workspace Protocol"

### PR DESCRIPTION
**Pull request by vrde**
The idea is to print this and put it on the door.

I was thinking about adding more rules, for example about headphones and stuff, but at the end I wanted to keep it minimal.

### CODE REVIEW

File: protocols/shared-workspace.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Goals
+SWP provides a reusable collaboration model for people working in the same workspace. It has these specific goals:
+
+1. To maximize focus and productivity, by reducing the sources of distraction.
+1. To reduce the burden to ask people to "behave" or "leave the room", by having an explicit protocol people agree before entering the room.
+
+## Design
+1. A person SHOULD NOT be loud.
+1. A person SHOULD NOT talk to more than one person at the time.
```
Comment (ttmc) [inline protocols/shared-workspace.md]: I think you meant to write, "more than one person at a time."

File: protocols/shared-workspace.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+1. One on one conversations MUST be short and occasional.
```
Comment (ttmc) [inline protocols/shared-workspace.md]: I'd write that as, "One-on-one conversations..."

File: protocols/shared-workspace.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+1. A person SHOULD write a message to another person before engaging in a conversation with them.
```
Comment (ttmc) [inline protocols/shared-workspace.md]: I'd move this rule first. Also, a "conversation" can be in text only, so maybe end with, "...before speaking with them."

File: protocols/shared-workspace.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+1. To maximize focus and productivity, by reducing the sources of distraction.
+1. To reduce the burden to ask people to "behave" or "leave the room", by having an explicit protocol people agree before entering the room.
+
+## Design
+1. A person SHOULD write a message to another person before speaking with them.
+1. A person SHOULD NOT be loud.
+1. A person SHOULD NOT talk to more than one person at a time.
+1. One-on-one conversations MUST be short and occasional.
+1. A person MUST NOT have meetings or calls unless everyone in the room is involved.
```
Comment (vrde) [inline protocols/shared-workspace.md]: I should remove "meetings". Technically, I can have a meeting on IRC. /cc @ttmc

Comment (ttmc) [reply protocols/shared-workspace.md]: Yeah, this rule is implied by the others, I think.

**Comment by vrde** [comment]
I'll move this to https://github.com/bigchaindb/beps.

**Comment by vrde** [comment]
I moved the PR to https://github.com/bigchaindb/BEPs/pull/22

**Pull request closed without merging**